spec/Features/user_management_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+
+feature 'User sign up' do
+	scenario 'requires a matching confirmation password' do
+
+	expect { sign_up(password_confirmation: 'wrong') }.not_to change(User, :count)
+end
```

Comment:
Lint/BlockAlignment: `end` at 7, 0 is not aligned with `scenario 'requires a matching confirmation password' do` at 4, 1.